src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetService.java

```diff
@@ -28,6 +29,25 @@
   Dataset createDataset(String datasetName, Country country, Language language,
       List<ByteArrayInputStream> records);
 
+  /**
+   * Creates a dataset id and publishes the given records for further processing
+   *
+   * @param datasetName must not be null
+   * @param country     must not be null
+   * @param language    must not be null
+   * @param records     must not be null
+   * @param xsltTransformerEDMExternal the xslt file as an input stream
```

Comment:
Forgot the name change I believe

Reply:
You are correct, I missed this change, I will updated it